### ISSUE TITLE
Solus 4.5 Xfce Fails to Hibernate

### ISSUE THREAD

**Issue by WetGeek** [getsolus/packages#1026]
### Summary

When I create any Vbox VM, I give it a swap partition of 6 GiB and RAM of 4 GB.  That should be all that's needed for it to be able to hibernate, as do all my hardware computers and 4.4 VMs.  But it's impossible for me to hibernate this Xcfe VM.  (I'm also having trouble hibernating a 4.5 Plasma VM, which was noted in a previous bug report.)

### Steps to reproduce

1. Create a VBox VM based on the Solus 4.5 Xfce RC1 .ISO.  Prepare its .VDI file with a 6 GiB swap partition.
2. Assign the VM 4 GB of RAM.
3. I spent some time configuring the VM as a daily-driver, similar to my laptop.
4. Attempt to cause the device to hibernate.  I used a `systemctl hibernate` command in terminology.

### Expected result

Normally the UI disappears, then reappears, then disappears once more when the session shuts down.

### Actual result

![image](https://github.com/getsolus/packages/assets/6805994/4317b1de-b6d0-4c77-a738-ae373c943255)

This is a rather absurd result, since the VM had 1 1/2 times the RAM size available as swap.

### Environment

- [X] Is system up to date?

### Repo

Unstable

### Desktop Environment

Xfce

### System details

![image](https://github.com/getsolus/packages/assets/6805994/55c3ad89-46bf-46f3-ab96-3f49823babb5)

**Comment by WetGeek** [comment]
I just installed RC2 on a DELL Latitude that just arrived today, and I provided that machine with a 24 GiB swap partition.  It has 16 GB of RAM.  So there's no reason for it to refuse to hybernate, but it does, and displays the exact error message shown above.  I.e., not enough swap space to hibernate.  Thus, it's not a VM issue.

**Comment by TraceyC77** [comment]
@WetGeek , I see you've opened a second issue for this bug - #1014 
Is there a need to keep both open? It seems like the information from the second can be added to this one, since they seem to be information about the same bug.

**Comment by WetGeek** [comment]
> @WetGeek , I see you've opened a second issue for this bug - #1014 Is there a need to keep both open? It seems like the information from the second can be added to this one, since they seem to be information about the same bug.

These were two _different_ problems, one real (the failure to restore a session on return from hibernation), and one the result of an error in creating a partition (the failure to begin a hibernation for lack of swap space).  On this laptop, the problem is the same as the one I reported for the VM.  

The hibernate command causes the computer to apparently perform all the right steps, but on return from hibernation the previous session is not restored.  That's real.  The workspaces, which all had contained an application, are all empty upon return.  It's as if it were a "reboot" instead of a "hibernate".  That's #1014, and it's quite valid.  And it exists on this laptop as well.

The bug where the hibernation couldn't begin because of not enough swap space was a simple error in creating the swap partition for this laptop.  It wasn't set to linux_swap as a format, so there wasn't enough swap space to perform the hibernation.  That's #1026, this one, and it should be closed.

This has resulted in some serious confusion for both of us, for which I'm sorry, but _this_ issue should be closed , and I'm going to re-check the issue I wrote up about the Plasma VM with the same symptoms (#1014).  That one is quite valid, but it may not be that it is just the Plasma VM that's affected, but all the VMs that I created from RC .isos.  Checking them all will take some time, but I want to see whether I should edit #1014 to apply to all the VMs that I created from the .isos, and not just Plasma.
